.github/workflows/javadoc.yml

```
@@ -0,0 +1,22 @@
+name: Upload javadocs to Github pages
+
+on: push
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout master branch
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Cache Maven packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+    - name: Refresh Javadoc
+      run: ./dev/docker.sh
```

```suggestion
      run: ./dev/javadoc.sh
```
Maybe this script instead of `docker.sh`?